### PR TITLE
[14.0][FIX] shopfloor_batch_automatic_creation: fix typo

### PR DESCRIPTION
**Pull request by nguyenhk99**

### CODE REVIEW

File: shopfloor_batch_automatic_creation/tests/test_batch_create.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def test_volume(self):
 
     def test_cluster_picking_select(self):
         self.menu.sudo().batch_create = True
-        self.menu.sudo().batch_create_max_pickings = 2
+        self.menu.sudo().batch_create_max_picking = 2
```
Comment (simahawk) [inline shopfloor_batch_automatic_creation/tests/test_batch_create.py:181]: seems legit, but: does it have any effect on tests? Seems not if it was passing before :thinking:

Comment (hailangvn) [reply shopfloor_batch_automatic_creation/tests/test_batch_create.py:181]: Thanks @simahawk , we are still scratching our head as we don't understand why the test was passing before AND the test was failed recently.

**Comment by simahawk** [comment]
@sebalix fine for you?

**Review by sebalix — APPROVED** [review]

**Pull request merged**